src/documentation/examples/Sections/FullWidthSliderSection.Images.slots.tsx

```diff
@@ -11,42 +11,44 @@ class App extends Vue {
         <Sections.FullWidthSliderSection
           onClick={console.log}
           scopedSlots={{
-            title: text => (
+            title: (text) => (
               <div class="ui-font-extrabold ui-text-2xl lg:ui-text-3xl xl:ui-text-5xl ui-italic">
                 {text}
               </div>
             ),
-            teaser: text => (
+            teaser: (text) => (
               <div class="ui-border-2 ui-rounded p-2">{text}</div>
             ),
-            button: props => (
+            button: (props) => (
               <button
                 class="Button Button--variant--inverted ui-rounded"
                 onClick={props.onClick}
               >
                 {props.content}
               </button>
             ),
-            media: slideMedia => (
+            media: (slideMedia: any) => (
```

Comment:
Why do we need the "any" type for the slideMedia, but not for the other attributes?